ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -158,26 +163,47 @@ export class Mint extends IronfishCommand {
       )
     }
 
+    const mintResponse = await client.mintAsset({
```

Comment:
There was a try/catch around this before. Do we want to keep that? Seems less likely that it will error without a post but still possible as we are contacting the node over the network

Reply:
you don't need a try / catch that's already in the underlying base command.